es-vue-base/src/lib-components/EsFooter.vue

```diff
@@ -15,100 +15,110 @@
+                                target="_blank">
+                                <span class="sr-only">{{ iconLink.text }}</span>
+                                <component :is="iconLink.icon" />
+                            </es-nav-bar-link>
+                        </div>  <!-- Social icons -->
+                    </div>  <!-- Logo and Social -->
+                </div>
+            </div>
+        </div>
+        <div class="bg-blue-700 text-blue-50 px-200 py-100">
+            <div class="container">
+                <!-- Trademark Info -->
+                <p class="font-size-75">
+                    {{ content.trademarkText }}
+                </p>
+                <p class="mb-200 font-size-75">
+                    {{ copyrightText }}
+                </p>
+                <!-- Trademark Info -->
+                <!-- Legal -->
+                <hr class="border-top bg-blue-50 m-0">
+                <div class="row justify-content-between pt-100">
+                    <div
+                        v-for="link in content.legalLinks"
+                        :key="link.text"
+                        class="col col-6 col-lg font-size-75 font-weight-semibold font-size-md-50 font-weight-md-normal mt-25 mb-50">
+                        <es-nav-bar-link
+                            :href="link.url"
+                            class="text-reset font-weight-normal">
+                            {{ link.text }}
+                        </es-nav-bar-link>
                     </div>
-                </div>  <!-- Link columns -->
-                <!-- Logo and Social-->
-                <div class="col col-12 col-lg-4 order-lg-0 mb-200">
-                    <es-nav-bar-link
-                        class="d-block mb-150"
-                        :href="content.home.link">
-                        <span class="sr-only">
-                            {{ content.home.name }}
-                        </span>
-                        <es-logo
-                            height="36px"
-                            width="160px" />
-                    </es-nav-bar-link>
-                    <!-- Social icons -->
-                    <div class="d-flex text-gray-700">
+                </div>  <!-- Legal -->
+            </div>
+        </div>
+        <div class="bg-blue-50 px-200 py-100">
+            <div class="container">
+                <!-- DOE -->
+                <div class="d-lg-flex align-items-end">
+                    <!-- eslint-disable-next-line -->
+                    <img
+                        :src="content.departmentOfEnergy.logoUrl"
+                        alt="department of energy logo"
+                        aria-hidden="true"
+                        class="mr-100 mb-100 mb-lg-0"
+                        width="99"
+                        height="24">
+                    <p class="mb-0 font-size-75 font-weight-normal font-weight-lg-semibold">
                         <es-nav-bar-link
```

Comment:
Can we apply `text-blue-900` to this to match the design?

Reply:
I tried adding `text-blue-900` but that doesn’t work because in `$link-color` in “_reboot.scss” is set to `blue-600`. To override that I added the custom styling, is there any different approach I could take?

Reply:
Hmm, are you applying it to the `es-nav-bar-link` component? It seems to work properly on my end when I do that along with removing the `style` tag at the bottom of the file.